data/categories.txt

```diff
@@ -2400,7 +2400,7 @@ da:3Metro|3undergrundsbane|3undergrund
 nl:3Metro
 fi:3Metro
 fr:3Métro
-de:2U-Bahn|3Metro|ÖPNV|4Ubahn
+de:3U-Bahn-Station|3Metro|ÖPNV|3Ubahnstation
```

Comment:
I would also suggest adding UBahn U-Bahn as a possible synonym for tourists and copy pastes. Did you test locally if typing `U-B` actually shows the hint?

Reply:
Also U-Bahn-Haltestelle

Reply:
> I would also suggest adding UBahn U-Bahn as a possible synonym for tourists and copy pastes. Did you test locally if typing `U-B` actually shows the hint?

No, I did not. 'U-B' does not work, unfortunately.

Reply:
- Unfortunately, the current search tokenizer will not work with synonyms like xx-yy-zz (and it doesn't work properly now for the current U-Bahn)
- Why U-Bahn is not enough here (of course if we assume a valid tokenizer)? Do we need to produce Station/Haltestelle mess?

Reply:
I propose to keep (2U-Bahn) here for the future search fix.

Reply:
>     * Why U-Bahn is not enough here (of course if we assume a valid tokenizer)? Do we need to produce Station/Haltestelle mess?

My thought was that the search term and the POI type in the search result should match. And since it will be suggested anyway, I thought it would be no problem to choose the exact term. But I can revert it if you want?

Reply:
Yes, revert back "2U-Bahn|3Metro|ÖPNV|4Ubahn"
This is way better for search.

Reply:
done